test/challenges-whitelist.json

```diff
@@ -0,0 +1,5 @@
+{
+      "http-01": [],
+      "tls-sni-01": [],
```

Comment:
Please add a non-empty list here to ensure that loading of such lists gets tested during integration testing.